code/modules/research/designs/weapon_designs.dm

```diff
@@ -78,7 +78,23 @@
 		/datum/material/iron = HALF_SHEET_MATERIAL_AMOUNT * 3,
 		/datum/material/bluespace = HALF_SHEET_MATERIAL_AMOUNT * 1.5,
 	)
-	build_path = /obj/item/ammo_box/magazine/m38/true
+	build_path = /obj/item/ammo_box/c38/true
+	category = list(
+		RND_CATEGORY_WEAPONS + RND_SUBCATEGORY_WEAPONS_AMMO
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_SECURITY
+
+/datum/design/c38_flare
+	name = "Speedloader (.38 Flare) (VERY Lethal)"
+	desc = "Designed to quickly reload revolvers. Flare casings launch a concentrated particle beam towards a target, lighting them up for everyone to see."
+	id = "c38_flare"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(
+		/datum/material/iron = HALF_SHEET_MATERIAL_AMOUNT * 3,
+		/datum/material/uranium = HALF_SHEET_MATERIAL_AMOUNT,
+		/datum/material/gold = HALF_SHEET_MATERIAL_AMOUNT,
+	)
+	build_path = /obj/item/ammo_box/c38/flare
```

Comment:
are you sure the TG version of this file has these changes?

Reply:
I let the incoming change from master go off on this one so it shouldn't be an issue